app/controllers/chugtypes_controller.rb

```diff
@@ -36,6 +36,7 @@ def create
   end
 
   def destroy
+    @chugtype.chugs.each { |chug| chug.destroy }
```

Comment:
In plaats van dit handmatig doen wil je waarschijnlijk `dependent: :destroy` gebruiken op de relatie in de models.

Reply:
Done 👍